src/sagas/clients.js

```diff
@@ -30,19 +31,21 @@ export function* exportProject({payload: {exportType}}) {
   const state = yield select();
   const project = getCurrentProject(state);
   const user = state.get('user').toJS();
-  let url;
+  let url, name;
 
   try {
     if (exportType === 'gist') {
       ({html_url: url} = yield call(createGistFromProject, project, user));
     } else if (exportType === 'repo') {
-      ({html_url: url} = yield call(createRepoFromProject, project, user));
+      ({url, name} = yield call(createRepoFromProject, project, user));
+    } else if (exportType === 'repo-update') {
```

Comment:
Would it be crazy to not treat this as a separate export type? In principle the saga here could decide which client method to call based on the presence of an existing repo name/URL in the project, or the client could just provide a `createOrUpdateRepoFromProject` which could decide what to do on the same basis. The message used for success is (sensibly) identical already, and the error messages could be merged with only minor modifications. WDYT?

Reply:
Got rid of `repo-export` all around - this actually got rid of a lot of duplication at the cost of a conditional in the `createOrUpdateRepoFromProject`